src/solver/optimisation/opt_appel_solveur_quadratique.cpp

```diff
@@ -36,13 +36,44 @@ extern "C"
 }
 
 #include <antares/logs/logs.h>
+#include "antares/optimization-options/options.h"
 #include "antares/solver/optimisation/opt_structure_probleme_a_resoudre.h"
+#include "antares/solver/utils/ortools_quadratic_wrapper.h"
+#include "antares/solver/utils/ortools_utils.h"
 
 using namespace Antares;
 
-bool OPT_AppelDuSolveurQuadratique(PROBLEME_ANTARES_A_RESOUDRE* ProblemeAResoudre,
+void SolveWithSirius(const Solver::Optimization::OptimizationOptions& options,
+                     PROBLEME_ANTARES_A_RESOUDRE* ProblemeAResoudre);
+void ProcessResult(PROBLEME_ANTARES_A_RESOUDRE* ProblemeAResoudre, int PdtHebdo);
+
+bool OPT_AppelDuSolveurQuadratique(const Solver::Optimization::OptimizationOptions& options,
```

Comment:
_(Not related to the PR but quick to change)_

**OPT_AppelDuSolveurQuadratique** takes **PdtHebdo** and pass it down to **ProcessResult** only to raise a warning.
We could remove this argument and make **ProcessResult** raise an exception. The client of **OPT_AppelDuSolveurQuadratique** could handle the warning.